src/main/java/com/example/DiceGameBE/model/Dice.java

```diff
@@ -0,0 +1,26 @@
+package com.example.DiceGameBE.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.springframework.data.redis.core.RedisHash;
+
+import java.io.Serializable;
+
+@Data
+@RedisHash
```

Comment:
W serwisie nie zapisujesz do bazy wyniku wylosowanych kości, jeżeli chcemy dodawać do bazy musimy to powiązać z grą  (klasa Game)